src/Codeception/Lib/Connector/Guzzle6.php

```diff
@@ -197,7 +204,11 @@ protected function doRequest($request)
         }
 
         try {
-            $response = $this->client->send($guzzleRequest, $options);
+            if (null !== $this->awsCredentials) {
+                $response = $this->client->send($this->awsSignature->signRequest($guzzleRequest, $this->awsCredentials), $options);
+            } else {
+                $response = $this->client->send($guzzleRequest, $options);
+            }
```

Comment:
I think that it would be good to copy this functionality to https://github.com/Codeception/Codeception/blob/2.3/src/Codeception/Lib/Connector/Guzzle.php
Guzzle connector is used when Guzzle 5 is installed (on PHP 5.4 with composer and on PHP 5.4-5.6 with phar file).

Reply:
Copied over to Guzzle v5